tests/fetchStats.test.js

```diff
@@ -253,4 +270,29 @@ describe("Test fetchStats", () => {
       rank,
     });
   });
+
+  it("should get commits of provided year", async () => {
```

Comment:
I removed the other test, which tested whether the current year is returned when no year is supplied. I did this because this is already tested in:

https://github.com/anuraghazra/github-readme-stats/blob/af6553ecf550fcbd6259b05294381f6cf2e126a2/tests/fetchStats.test.js#L127

https://github.com/anuraghazra/github-readme-stats/blob/af6553ecf550fcbd6259b05294381f6cf2e126a2/tests/fetchStats.test.js#L216

and

https://github.com/anuraghazra/github-readme-stats/blob/af6553ecf550fcbd6259b05294381f6cf2e126a2/tests/fetchStats.test.js#L191

Reply:
@anuraghazra are you okay with this implicit testing or do you want to have a explicit test like @vzsky did in 

https://github.com/anuraghazra/github-readme-stats/blob/4b15721f691d097919f56494d988034532da0790/tests/fetchStats.test.js#L193

Reply:
I did that just to be sure that the mocked function works fine. I think deleting that is probably okay since the mocked function was tested and is not likely to get changed.